modules/installation-osp-creating-network-resources.adoc

```diff
@@ -74,11 +74,11 @@ See "Enabling access to the environment" for more information.
 $ ansible-playbook -i inventory.yaml security-groups.yaml
 ----
 
-. On a command line, create a network, subnet, and router by running the `network.yaml` playbook:
+. On a command line, create a network, subnet, router, and ports by running the `update-network-resources.yaml` playbook:
```

Comment:
That's statement is not accurate. This playbook will add tags to network, subnet(s), ports and router. It will also attach floating IP to API and Ingress ports and sets security groups on those ports.

Reply:
I made this change as part of the new 'creating-network-resources' file. I added the details as a callout. 
![image](https://github.com/openshift/openshift-docs/assets/46728411/9558c749-ab65-4808-9677-79669eeeb222)